charts/config/templates/uds-package.yaml

```
@@ -268,7 +268,9 @@ spec:
         remoteNamespace: {{ .Values.redis.namespace | quote }}
         remoteSelector:
           {{ .Values.redis.selector | toYaml | nindent 10 }}
-        port: {{ .Values.redis.port }}
+        ports:
+          - {{ .Values.redis.port }}
+          - {{- if .Values.redis.sentinel.enabled }}{{ .Values.redis.sentinel.port }}{{- end }}
```

```suggestion
          {{- if .Values.redis.sentinel.enabled }}
          - {{ .Values.redis.sentinel.port }}
          {{- end }}
```
the dash will cause invalid yaml when sentinel is disabled

---

ah, good catch